test/snippet/range/views/minimiser.cpp

```diff
@@ -18,4 +19,22 @@ int main()
 
     // kmer_hash with gaps, hashes: [2,7,8,14,1], minimiser: [2,1]
     seqan3::debug_stream << (text | seqan3::views::kmer_hash(0b101_shape) | seqan3::views::minimiser(4)) << '\n';
+
+    /* Minimiser view with two ranges
+     * The second range are the hash values from the reverse complement, the second reverse is necessary to put the
+     * hash values in the correct order. For the example here:
+     * ACGTAGC | seqan3::views::complement                      => TGCATCG
+     *         | std::views::reverse                            => GCTACGT
+     *         | seqan3::views::kmer_hash(seqan3::ungapped{3})  => [39 (for GCA), 28 (for CTA), 49 (for TAC),
+     *                                                              6 (for ACG), 27 (for CGT)]
+     * "GCA" is not the reverse complement from first k-mer in "ACGTAGC", which is "ACG", but "CGT" is.
+     * Therefore, a second reverse is necessary to find the smallest value between the original sequence and its
+     * reverse complement.
+     */
+    auto hashes2 = text | seqan3::views::complement | std::views::reverse
+                        | seqan3::views::kmer_hash(seqan3::ungapped{3}) | std::views::reverse;
+    seqan3::debug_stream << hashes2 << '\n'; // [27,6,49,28,39]
+
+    auto minimiser2 = hashes | seqan3::views::minimiser(4, hashes2);
+    seqan3::debug_stream << minimiser2 << '\n'; // [6,6]
```

Comment:
I thought that the same hash values are skipped.

Reply:
But it is not the same hash value, it is 6 from first k-mer in ACGTAGC and 6 from second kmer in the reverse complement, so from CGT in  ACGTAGC